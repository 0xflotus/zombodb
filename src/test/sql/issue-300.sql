select e.id, event_type, zdb.score(u.ctid) <> 0 from events e, users u where e.user_id = u.id and event_type = 'IssueCommentEvent' and u ==> 'vicjoecs' order by zdb.score(u.ctid) desc, id;

select sum(zdb.score(ctid)) > 0 from events where events ==> 'beer';
select sum(zdb.score(ctid)) > 0, event_type from events where events ==> 'beer' group by event_type order by event_type;