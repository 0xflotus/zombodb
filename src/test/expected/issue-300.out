select e.id, event_type, zdb.score(u.ctid) <> 0 from events e, users u where e.user_id = u.id and event_type = 'IssueCommentEvent' and u ==> 'vicjoecs' order by zdb.score(u.ctid) desc, id;
   id   |    event_type     | ?column? 
--------+-------------------+----------
 122357 | IssueCommentEvent | t
 123707 | IssueCommentEvent | t
 123783 | IssueCommentEvent | t
(3 rows)

select sum(zdb.score(ctid)) > 0 from events where events ==> 'beer';
 ?column? 
----------
 t
(1 row)

select sum(zdb.score(ctid)) > 0, event_type from events where events ==> 'beer' group by event_type;
 ?column? |    event_type     
----------+-------------------
 t        | IssueCommentEvent
 t        | IssuesEvent
 t        | CreateEvent
 t        | ReleaseEvent
 t        | PushEvent
(5 rows)

